examples/grafana-integration/datasources.yaml

```diff
@@ -0,0 +1,18 @@
+ApiVersion: 1
```

Comment:
is this the same as examples/grafana-integration/grafana/provisioning/datasources/datasources.yaml?

Reply:
Yes, it was a duplicate, now it's deleted.

Reply:
I still see the other file, which looks identical